policy api cannot store large policies

storing large policies fails with the below exception.

Caused by: org.h2.jdbc.JdbcSQLDataException: Value too long for column "POLICY CHARACTER VARYING(255)": "U&'package example\\000d\\000aimport future.keywords # we want ""every"" and ""in""\\00... (1334)"; SQL statement:
insert into policies (create_time,csp,enabled,last_modified_time,policy,user_id,id) values (?,?,?,?,?,?,?) [22001-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:506)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.getValueTooLongException(DbException.java:322)

--- a/modules/database/src/main/java/org/eclipse/xpanse/modules/database/policy/PolicyEntity.java
+++ b/modules/database/src/main/java/org/eclipse/xpanse/modules/database/policy/PolicyEntity.java
@@ -48,6 +48,7 @@ public class PolicyEntity extends CreateModifiedTime {
     /**
      * The valid policy created by the user.
      */
+    @Column(name = "POLICY", length = Integer.MAX_VALUE)
     private String policy;
 
     /**

